source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -933,6 +965,27 @@ export class CardDesignerSurface {
         }
     }
 
+    reassignCardElementToCarousel(carouselPeer: DesignerPeers.CarouselPeer) {
+
+        const carouselPage = carouselPeer.getChildAt(0);
+
+        if (carouselPage instanceof DesignerPeers.CardElementPeer) {
+
+            // Can't modify the list as we loop through it
+            const children = [];
+
+            for (let child of this._rootPeer.children) {
+                children.push(child);
+            }
+
+            for (let child of children) {
+                if (!(child instanceof DesignerPeers.CarouselPeer)) {
+                    carouselPage.tryAdd(child);
```

Comment:
Add a comment to code like `tryAdd remove element from the children, then append it to the last (possible wrong description here). and this function use this feature to shuffle the order`